stellargraph/core/utils.py

```diff
@@ -218,5 +170,7 @@ def preprocess_adj(adj, symmetric=True):
                     type(k), k
                 )
             )
+    elif method == "chebyshev":
+        raise ValueError("method 'chebyhev' did not behave correctly and has been removed")
```

Comment:
Err oops:

```suggestion
        raise ValueError("method 'chebyshev' did not behave correctly and has been removed")
```

Also, the overall diff will be slightly cleaner if you move this above `sgc`, so that it's where it was before.